src/Nethermind/Nethermind.Trie/TrieNode.cs

```diff
@@ -111,6 +116,50 @@ void ThrowAlreadySealed()
                 }
             }
         }
+        public ref readonly CappedArray<byte> ValueRef
```

Comment:
this is very similar to `Value.get`,  can we make the code reusable? (Maybe Value getter using this and dropping ref?)

Reply:
👍